about.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>About Creative-Hub Summit</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="Design/css/style.css">
+  </head>
+  <body>
+    <header class = "shared_header">
+      <!--Dynamic header content loaded from JS DOM-->
+    </header>
+    <main class="parent_main">
+        <section class="about_header">
+          <div class="welcome-text col-12 pt-5 mt-5">
+            <h3 class="text-left text-center hello_intro orange">"Home for Creatives!"</h3>
+            <h1 class="text-uppercase text-center text-md-center font-weight-bold text-img-effect">SOFTWARE DEVELOPERS ETHIOPIA SUMMIT 2022</h1>
+          </div>
+          <div class="about_desc col-12 mt-4 mb-4">
+            <p class="bg-white text-center p-4">
+              The Creative Hub Ethiopia initiated by UNIDO and FeSMMIPA with funding 
+              from the Italian Development Cooperation. Iceaddis and African Mosaique are 
+              implementing partners.We offer a wide range of services to designers and 
+              innovative entrepreneurs mainly to help reinforce and enhance creativity through 
+              various focused and thematic technical support, innovation seminars and workshops, 
+              training, prototyping techniques, social events, and networking sessions. 
+              The Hub also has a fully equipped digital library, rapid prototyping services (e.g 3D printing), 
+              mock-up labs, meeting rooms, space for exhibition and events, co-working space, and cafeteria.
+            </p>
+          </div>
+          <div class="event-date col-12 pb-3">
+            <p class="text-center w-50 m-auto pb-3">
+              Please contact us per Email for any further questions
+              about Software Developer's Conference Summit
+              <span class="font-weight-bold pb-4">
+                <u>Abenezermesafint2128@gmail.com</u>
```

Comment:
- [ ] Please take a look at your email here. On large screens (mobile) everything is fine, but when you go smaller, it does not show as expected. Again, probably the problem has to see with the sizes given to the containers, but please take a look to try to fix it! 
Here's another image for reference:
- If you go smaller than 370px the email is not showing completely, which can lead in a bad user experience.
![Screenshot_2022-02-09_19-15-04](https://user-images.githubusercontent.com/59100477/153265382-2bbbc138-e81e-45a5-a5db-71104f91887a.png)

Reply:
Done.